README.md

```diff
@@ -10,7 +10,7 @@ When the Zipwhip landline receives an incoming message ZipChat performs the foll
 
 ### Keywords
 
-JOIN [username] [channelname]
+JOIN [channelname] [username]
```

Comment:
We don't support passing in user at all. The user is implied by the display name of the joining user, as configured in the setup data (Mongo).